docker/docker-entrypoint.sh

```diff
@@ -60,16 +60,25 @@ fi
 # assign default identity dir if not set in parent env; this is a writeable path within the container image
 : "${ZITI_IDENTITY_DIR:="/ziti-edge-tunnel"}"
 
+# approximates the functionality of the mountpoint command, which could be installed with util-linux at the expense of 20MB additional image footprint
+if [ "$(stat -c %d:%i "${ZITI_IDENTITY_DIR}")" = "$(stat -c %d:%i "$(dirname "${ZITI_IDENTITY_DIR}")")" ]; then
```

Comment:
this doesn't feel like the place to put this. ZET should be doing this check and printing the warning

Reply:
That wouldn't work in this case because there's no way to know it's running in a container which is the only time a mountpoint is applicable.

Reply:
I don't think we're on the same page. if the ZET does it, it's irrelevant where it runs. The directories/files either would be, or would not be writable. I don't understand how running in a container matters